tests/kola/misc-ro/misc-ro.sh

```diff
@@ -132,3 +132,10 @@ if ! test -f /etc/iscsi/initiatorname.iscsi; then
     fatal "Missing /etc/iscsi/initiatorname.iscsi"
 fi
 echo "ok iSCSI initiator name"
+
+
+# Let's make sure the NetworkManager we use is the one we're expecting
+if [ $(rpm -q NetworkManager) != "NetworkManager-1.26.0-12.1.rhaos4.7.el8.x86_64" ]; then
```

Comment:
Could we just check for `rhaos4.7` instead so that we don't have to update the pin each time, but we still make sure that we're not pulling in the `baseos` one? (That's what we really care about, right?)

Reply:
yeah, I honestly wanted it to be a little more stringent just so we could make sure things didn't go awry (while we're on 8.3), but I could see relaxing the check just for `rhaos4.7`.

Reply:
new upload with relaxed check